source/MaterialXCore/Document.h

```diff
@@ -568,6 +591,27 @@ class ScopedUpdate
     DocumentPtr _doc;
 };
 
+/// @class @ScopedDisableNotifications		
```

Comment:
I would suggest "ScopedSuspendCallbacks" for the name of this class.

Reply:
As stated in my general comment, this class is not needed anymore.

Reply:
I can make this change. See my general comment below.